src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.TestFramework/KeyVaultTestFixture.cs

```diff
@@ -127,6 +132,8 @@ private bool FromConfiguration()
             string authSecret = TestConfigurationManager.TryGetEnvironmentOrAppSetting("AuthClientSecret");
             string standardVaultOnlyString = TestConfigurationManager.TryGetEnvironmentOrAppSetting("StandardVaultOnly");
             string softDeleteEnabledString = TestConfigurationManager.TryGetEnvironmentOrAppSetting("SoftDeleteEnabled");
+            string storageUrl1 = TestConfigurationManager.TryGetEnvironmentOrAppSetting("StorageResourceUrl1");
```

Comment:
Something to keep in mind for next PR
@rijethma the latest version of testFramework is now capable of honoring custom key value pairs. This will help in not hard coding custom data or use additional settings file to inject custom data for your tests.
As long you provide your custom key=value pair in connection string.
At runtime you should be able to retrieve it using
TestEnvironment.ConnectionString.KeyValuePairs["KeyName"]